README.md

```diff
@@ -147,21 +163,42 @@ DID: `did:nft:{chainNamespace}.{chainReference}_erc1155.{contractAddress}_{token
 
 CAIP-29: `{chainNamespace}:{chainReference}/erc1155:{contractAddress}/{tokenId}`
 
-
 ### Conversions
+
 **DID->CAIP**
+
 ```
 const caip = did.substr(8).replace(/_/g, '/').replace(/\./g, ':')
 ```
+
 **CAIP->DID**
+
 ```
 const did = `did:nft:${caip.replace(/\//g, '_').replace(/:/g, '.')}`
 ```
 
+There are helpers that help you with the conversion:
+
+```typescript
+import { caipToDid, didToCaip } from 'nft-did-resolver'
+import { AssetId } from 'caip'
+
+// CAIP -> DID URL
+const assetId = new AssetId(
+  AssetId.parse('eip155:1/erc721:0x1234567891234567891234567891234596351156/0x1')
+)
+const didUrl = caipToDid(assetId) // did:nft:eip155.1_erc721.0x1234567891234567891234567891234596351156_0x1
+const didUrlWithTimestamp = caipToDid(assetId, 1628529680) // did:nft:eip155.1_erc721.0x1234567891234567891234567891234596351156_0x1?versionTime=2021-08-09T17:21:20Z
```

Comment:
Would be nice if devs don't need to know about `caip` at all, e.g. I would prefer a helper function like this:

```ts
const didUrl = createNftDIDUrl({
  chainId: 'eip155:1',
  asset_namespace: '0x1234567891234567891234567891234596351156',
  asset_id: '0x01'
})
```

Basically this would just use the [caip constructor](https://github.com/pedrouid/caip-js/blob/master/src/assetId.ts#L31) (accepts both json & string) then prepend `did:nft` and replace `/` with `_`.

Reply:
_Added_ `createNftDidUrl` that does not rely on `AssetId` as input parameter. I think it makes sense still to retain helper to convert `caip.AssetId`.